src/components/Button.js

```diff
@@ -22,10 +25,19 @@ const Button = styled.button`
   }
 `;
 
-export const ListButton = ({ title, onClick }) => {
+export const ListButton = ({ title, audiofile }) => {
+  const [playing, setPlaying] = useState(false);
+
+  const toggle = () => setPlaying(!playing);
+
+  useEffect(() => {
+    const audioElement = new Audio(audiofile);
+    playing ? audioElement.play() : audioElement.pause();
+  }, [playing, audiofile]);
+
```

Comment:
with every calling of useEffect, you create a new audioElement 
```suggestion

     const audioElement = new Audio(audiofile);
  
    useEffect(() => {
    playing ? audioElement.play() : audioElement.pause();
  }, [playing, audiofile]);

```

this way there is only one reference to the audiofile which the function refers to

Reply:
THX!